### PR TITLE
Feature 1 / Log In and Sign Up design

### LINKED ISSUE #5: make the sign in screen

### LINKED ISSUE #6: make the login screen

### PR DESCRIPTION
**Pull request by HugoCamposLopez**
# Description
The design of Log In and Sign Up screens are finished.
 
 # Testing
We have tested the checkbox feature.
 
  
  ## Screenshots 
If applicable, include screenshots of the results or screenshots that help to see changes
image
image

### CODE REVIEW

**Comment by Namoku** [comment]
Following [this](https://codeclimate.com/github/BC-MAY-21-RN/kata-05-reservacion-de-vuelos-k5-n21-t5/pull/7), you can refactor the parts easily, for example:
```js
<FormInput
  style={Styles.textInput}
  labelVal={password}
  secureTextEntry={true}
  onChangeText={(userpass) => setPassword(userpass)}
```
into a new component it can returns this:
```js
const ExampleNewComponent = () => {
return (
<FormInput
  style={Styles.textInput}
  labelVal={password}
  secureTextEntry={true}
  onChangeText={(userpass) => setPassword(userpass)}
....etc....
```
and you'll solve a lot of issues those you have in the code review.

File: ios/kata05.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -12,6 +12,22 @@
...
+		66E5F665CF774CCC9BCBE10D /* Fontisto.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 1CC8CC5CE38443CC99419D88 /* Fontisto.ttf */; };
+		7D128670442240F3AA807B3E /* Foundation.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 153B32382F2E4463A38984CD /* Foundation.ttf */; };
+		942A027894584394847C95B4 /* Ionicons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = E2BB729D95DB40CE8DB98C04 /* Ionicons.ttf */; };
+		141EB20F87764FFDB9321C87 /* MaterialCommunityIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = D8E8D2B0A9D24A7AA67B7BA2 /* MaterialCommunityIcons.ttf */; };
+		112765ACA5AF44B9A1E14175 /* MaterialIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = C94AB4ACBF324DCCA4955E87 /* MaterialIcons.ttf */; };
+		228A2758F3D849D4928ADC29 /* Octicons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = C4FF0E88E3AC4E32B9D87AA7 /* Octicons.ttf */; };
+		8DCCA2A7F2A64F0CBA4BAF32 /* SimpleLineIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = C542EAADECAB4F5F98AC3612 /* SimpleLineIcons.ttf */; };
+		C8FDF27E203A48E5837F185C /* Zocial.ttf in Resources */ = {isa = PBXBuildFile; fileRef = CA42A6B2B63F44B5B483936A /* Zocial.ttf */; };
```
Comment (YilKanda) [inline ios/kata05.xcodeproj/project.pbxproj]: Why are you adding multiple fonts?

**Review by YilKanda — CHANGES_REQUESTED** [review]
My observation in this PR is that you are installing too many typefaces. It is advisable to have a guideline of typefaces to use. Example, I leave you the link of [FontJoy](https://fontjoy.com/), where you can make a combination of 3 fonts. But remember that the combination of many typographic fonts only incites confusion and is not intuitive for Internet users, in addition the use of a certain font is also essential since each one has a personality and transmits a message.

I imagine all these fonts are added by icons, but do you really take up all the icons from those fonts?

File: ios/kata05.xcodeproj/project.pbxproj (hunk 1 above)
Comment (HugoCamposLopez) [reply ios/kata05.xcodeproj/project.pbxproj]: We didn't notice that. 
We installed Vector Icons and it should be added automatically.

**Comment by HugoCamposLopez** [comment]
> Following [this](https://codeclimate.com/github/BC-MAY-21-RN/kata-05-reservacion-de-vuelos-k5-n21-t5/pull/7), you can refactor the parts easily, for example:
> 
> ```js
> <FormInput
>   style={Styles.textInput}
>   labelVal={password}
>   secureTextEntry={true}
>   onChangeText={(userpass) => setPassword(userpass)}
> ```
> 
> into a new component it can returns this:
> 
> ```js
> const ExampleNewComponent = () => {
> return (
> <FormInput
>   style={Styles.textInput}
>   labelVal={password}
>   secureTextEntry={true}
>   onChangeText={(userpass) => setPassword(userpass)}
> ....etc....
> ```
> 
> and you'll solve a lot of issues those you have in the code review.

Thank you, we just did that, and now we have less code lines.

File: src/components/checkBox/CheckBox.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    </View>
+  )
+}
+
+
+
+
+
+
+
```
Comment (YilKanda) [inline src/components/checkBox/CheckBox.js]: Eliminate unnecessary spaces.

File: src/components/checkBox/CheckBoxStyle.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        fontSize: 10,
+        margin: 8,
+        marginLeft: 0
+    },
+    textUnderLine:{
+        textDecorationLine: 'underline'
+    }
+})
+
+export default CheckBoxStyle
```
Comment (YilKanda) [inline src/components/checkBox/CheckBoxStyle.js]: Add a blank line.

File: src/components/googleButton/GoogleButton.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+          <Image style={GoogleStyles.imageGoogle} source={{uri: 'http://assets.stickpng.com/images/5a951939c4ffc33e8c148af2.png'}}/>
+        </View>
+
+        <View >
+          <Text style={GoogleStyles.buttonText}>{buttonTitle}</Text>
+        </View>
+
+    </TouchableOpacity>
+  )
+}
```
Comment (YilKanda) [inline src/components/googleButton/GoogleButton.js]: Add a blank line.

File: src/components/googleButton/GoogleButtonStyle.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+  },
+  imageContainer:{
+    width: '17%',
+  },
+  imageGoogle:{
+    width: 30,
+    height: 30,
+  }
+});
```
Comment (YilKanda) [inline src/components/googleButton/GoogleButtonStyle.js]: Add a blank line.

File: src/components/index.js [hunk 6]
```diff
@@ -1 +1,7 @@
-export {CheckBox} from './CheckBox'
+export {FormInput} from './Forminput/FormInput'
+export {FormButton} from './FormButton/FormButton'
+export {CheckB} from './checkBox/CheckBox'
+export {GoogleButton} from './googleButton/GoogleButton'
+export {InputPassword} from './Forminput/InputPassword'
+export {InputEmail} from './Forminput/InputEmail'
+export {Footer} from './Footer/Footer'
```
Comment (YilKanda) [inline src/components/index.js]: Add a blank line.

File: src/helpers/size.js [hunk 7]
```diff
@@ -0,0 +1,3 @@
+import {Dimensions} from 'react-native';
+export const windowWith = Dimensions.get('window').width;
+export const windowHeight = Dimensions.get('window').height;
```
Comment (YilKanda) [inline src/helpers/size.js]: Add a blank line.

File: src/screens/login.js [hunk 8]
```diff
@@ -1,27 +1,44 @@
...
+import { SafeAreaView } from "react-native-safe-area-context"
+import { InputPassword, FormButton, InputEmail } from '../components/index'
 import Styles from '../screens/style/common_style'
 
+
+
 export const login = (props) => {
+  const {navigation} = props
+  return (
+
```
Comment (YilKanda) [inline src/screens/login.js]: Remove the unnecessary spaces.

**Review by YilKanda — COMMENTED** [review]
Before uploading your *.js files, I recommend you make use of [ESLint](https://eslint.org/docs/user-guide/getting-started), to verify the formatting of your code and, thus, keep it as readable as possible.

There are many unnecessary spaces in your files.

Also, remember to put a blank line at the end of each file.

File: src/screens/login.js (hunk 8 above)
Comment (HugoCamposLopez) [reply src/screens/login.js]: We have done what you asked and we deleted the fonts that we are not using.

**Review by YilKanda — COMMENTED** [review]
Check the indentation of your files.

**Review by YilKanda — APPROVED** [review]

**Pull request merged**